README.md

```diff
@@ -98,149 +98,142 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
- - Fixed commonly reported PowerToys Run startup crashes after an upgrade.
- - New settings and GPO policies to help control behavior after an upgrade. Thanks [@htcfreek](https://github.com/htcfreek)!
-
-Here are some screenshots of the new languages:
-![Arabic SA Settings screenshot](https://github.com/microsoft/PowerToys/assets/26118718/be27096d-6c03-4b09-afc4-478ca427e3ec)
-![Hebrew Settings screenshot](https://github.com/microsoft/PowerToys/assets/26118718/e1435060-1f94-4e41-adee-1d0a609584ca)
-![Persian Settings screenshot](https://github.com/microsoft/PowerToys/assets/26118718/8592dcb7-8a04-4831-9325-a8b9b05787df)
-![Ukrainian Settings screenshot](https://github.com/microsoft/PowerToys/assets/26118718/24242dd8-eb17-4859-b2e4-1e5c63ffbffd)
+ - New feature: Keyboard Manager allows mapping shortcuts to start applications or opening URIs. Thanks [@jefflord](https://github.com/jefflord)!
+ - New feature: Keyboard Manager allows shortcuts with chords. Thanks [@jefflord](https://github.com/jefflord)!
+ - Modernized Color Picker with Fluent UX. Thanks [@niels9001](https://github.com/niels9001)!
+ - Peek now is able to preview drives. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - File Locksmith has a Windows 11 tier 1 context menu.
 
 ### General
 
- - Added Arabic (Saudi Arabia) translation.
- - Added Hebrew translation.
- - Added Persian translation.
- - Added Ukrainian translation.
- - Improved the file watcher used across many utilities to consume less resources. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Refactored code so that the English language is used as a fallback when a localized resource is not found.
+
+### Awake
 
-### AlwaysOnTop
+ - The setting reverts to "Keep using the current power plan" after Awake deactivates itself after any of the timed modes has expired.
 
- - Fixed an invisible border issue when the border color was set to the black color.
- - Added the AlwayOnTop icon to the base application executable. Thanks [@ckirby19](https://github.com/ckirby19)!
+### Color Picker
+
+ - Now uses WPFUI and the UI was updated to follow Fluent UX principles. Thanks [@niels9001](https://github.com/niels9001)!
+ - Added enable and disable telemetry to align it with the other utilities.
 
 ### Command Not Found
 
- - Signed the PowerShell scripts used by the Command Not Found installation process.
+ - Added telemetry for when a module instance is created in PowerShell.
+
+### FancyZones
+
+ - Fixed a memory leak occurring on work area changes.
 
 ### File Explorer add-ons
 
- - Fixed an issue causing SVG Thumbnail generation to hang when trying to preview SVG files at the same time.
+ - Added support to the .ksh, .zsh, .bsh and .env file types to Monaco previewer. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+ - Re-enabled the RendererAppContainer feature in WebView 2, since the associated crash has been fixed in the latest WebView 2 releases.
```

Comment:
```suggestion
 - Re-enabled the RendererAppContainer feature in WebView2, since the associated crash has been fixed in the latest WebView2 releases.
```